templates/gax/python/tox.ini

```diff
@@ -15,8 +15,9 @@ deps =
 [testenv:upload-package]
 basepython = python2.7
 commands =
+  rm -rf grpc_google_pubsub_v1.egg-info/ dist/ setup.cfg
   python -c 'import os; os.rename("MANIFEST.in", "MANIFEST.in.orig")'
-  python -c 'open("MANIFEST.in", "w").write(open("MANIFEST.in.orig").read() + "\nglobal-exclude *_pb2.py\n")'
+  python -c 'open("MANIFEST.in", "w").write(open("MANIFEST.in.orig").read() + "global-exclude *_pb2.py\n")'
```

Comment:
By removing the newline, we assumes that "MANIFEST.in.orig" file will always ends with a newline. Would that cause some issue?